src/cpuid/src/common.rs

```diff
@@ -147,4 +215,46 @@ pub mod tests {
         assert!(vendor_id.is_ok());
         matches!(&vendor_id.ok().unwrap(), VENDOR_ID_INTEL | VENDOR_ID_AMD);
     }
+
+    #[test]
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    fn test_is_same_model() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let vcpu = vm.create_vcpu(0).unwrap();
+
+        let mut curr_cpuid = vcpu.get_cpuid2(KVM_MAX_CPUID_ENTRIES).unwrap();
```

Comment:
I would create the cpuid manually instead of calling `vcpu.get_cpuid2()`. You can find examples in other unit tests.

Reply:
Done.